translations/ui-courses/en.json

```diff
@@ -192,5 +192,10 @@
     "addFastAddItem.title": "Add Fast Add item",
     "addFastAddItem.tooltip": "Create a new Inventory record using 'Fast Add' and add it to this course as a reserve",
     "errors.mustBeInteger": "This value must be an integer",
-    "SENTINEL": "SENTINEL"
+    "SENTINEL": "SENTINEL",
+    "cv.patronGroup.cannotDeleteTermHeader": "Cannot delete the patron group",
```

Comment:
The use of `patronGroup` in the names of these translations cannot possible be right.

Reply:
Removed  `patronGroup` from key.